common/app/model/dotcomrendering/pageElements/PageElement.scala

```diff
@@ -198,14 +198,15 @@ object PageElement {
       ))
 
       case Image =>
+
         def ensureHTTPS(src: String): String = src.replace("http:", "https:")
 
         val signedAssets = element.assets.zipWithIndex
           .map { case (a, i) => ImageAsset.make(a, i) }
 
         val mediaType = (hasShowcaseMainElement, isImmersive) match {
-          case (true, _) => ImmersiveMedia
-          case (_, true) => MainMedia
+          case (true, _) => MainMedia
```

Comment:
Definitely not `Showcase` as below? (just checking)

Reply:
To be honest, yes and no. There are two improvements I need to (and plan to) do. The first is to write the proper function to decide both the correct media type and the correct role, and for this I currently don't know which inputs those functions should use (I currently use `hasShowcaseMainElement` and `isImmersive` to mirror DCR's current complexity but that will change). The second is to see how I can apply those per element not driven by page attributes. 

Just for this patch, the choice of `MainMedia` was meant to provide big enough pictures without affecting too much the other pictures of the same page... (Trying to be conservative in the size of pictures...). This is definitively a work in progress :)